library/Exceptions/FalseValException.php

```diff
@@ -20,10 +20,10 @@ final class FalseValException extends ValidationException
      */
     protected $defaultTemplates = [
         self::MODE_DEFAULT => [
-            self::STANDARD => '{{name}} is not considered as "False"',
+            self::STANDARD => '{{name}} must be similar to `false`',
```

Comment:
I think _ evaluate to_ is better. In case of a form, any message that includes "false" would suck, anyways. At least it's a bit more clear what this means.

I will make those changes, and I will merge it then.